FogCreek/Glitch-Community
# Sugary anthropology: componentize new stuff modal

## Links
* Remix link: https://sugary-anthropology.glitch.me/

## Changes:
* componentize NewStuff modal

## How To Test:
* log into https://sugary-anthropology.glitch.me/
* click the new stuff puppy
* it should open the new stuff modal
* tabbing should only focus elements in the modal
* clicking the background or keying `escape` should clear the modal
